Driver/Video/VGAlike/VGA16/vga16Tables.asm

```diff
@@ -198,6 +200,8 @@ ifndef PRODUCT_WIN_DEMO
 		nptr    offset VidSetVESA		; VD_VESA_800x480_16
 		nptr    offset VidSetVESA		; VD_VESA_832x624_16
 		nptr    offset VidSetVESA		; VD_VESA_848x480_16
+		nptr    offset VidSetVESA		; VD_VESA_960x540_16

```

Comment:
I think it is an leaning from the last weeks, that it is helpful to increase the "change" number of the geode version at least if some property of a geode changes, as new entries in the list of devices. This will cause the cache in the device driver list to be rerendered so the new entries become available automatically.